exercises/diffie-hellman/src/test/java/DiffieHellmanTest.java

```diff
@@ -19,12 +19,15 @@ public void setUp() {
 
     @Test
     public void testPrivateKeyInRange() {
-        BigInteger primeP = BigInteger.valueOf(23);
+        BigInteger prime = BigInteger.valueOf(23);
         List<BigInteger> privateKeys = new ArrayList<>();
-        for (int i = 0; i < 10; i++) privateKeys.add(diffieHellman.privateKey(primeP));
+        for (int i = 0; i < 10; i++) {
+		privateKeys.add(diffieHellman.privateKey(prime));
```

Comment:
I think the indentation has gone slightly wrong here? This should be indented four spaces in from the `for` statement and the closing curly bracket should be indented the same as the `for` statement.

The second `assertTrue` statement in the next for loop has the same problem, as well as the for loop in `testPrivateKeyRandomlyGenerated` 🙂

Reply:
Sorry, I didn't notice it. Fixed! 🙂